developers/contributing_how_tos.md

```diff
@@ -0,0 +1,41 @@
+# How to contribute a napari How-To
+
+This should be a sentence (or brief paragraph) about what this how-to will achieve and how it helps address a common problem/task/need.
+
+## Prerequisites
+
+- this template
+- a list of what the user needs to prepare to be ready to start this how-to guide 
+- links to any existing napari.org tutorials or how-to guides that can help the user fulfill these prerequisites
+- the level of python/napari knowledge required to follow this how-to
+    - try to be specific about what skills are needed e.g. 
+    'connecting callbacks to layer events' or 'using matplotlib to produce plots'
+- plugins that should be installed
+- python packages that should be installed (don't list napari dependencies)
+
+***
+
+## How-to steps
+
+Headings in this section are yours to decide on. You should split your how-to guide into easy to follow, sequential steps to achieve the stated goal. Remember that your how-to may be used as a reference, so your headings should be descriptive and allow users to get a quick overview of the document, and which steps are relevant to them.
```

Comment:
I like the idea of having an `outline` section before the actual steps in the how-to document. This section would quickly lists the steps, providing a super short summary of the steps to be taken and, if appropriate, a little more detail than the title of the section.

Reply:
I think this will be automatically generated by the headings in the right-side toolbar, so not sure we need to include this again in the document itself.

Reply:
@melissawm you're right - I was initially thinking that for a longer form how-to this could be useful for providing a bit more detail but actually how-tos should probably remain very limited in scope

Reply:
@alisterburt I think that sort of outline/longer form intro would be perfect for a tutorial, but would expect how-tos to have limited enough scope for a brief paragraph to be sufficient (if a sentence isn't quite).